reference/format/datagrid.rst

```diff
@@ -0,0 +1,166 @@
+Datagrid
+========
++-----------+------------------+
+| Filename  | ``datagrid.yml`` |
++-----------+------------------+
+| Root Node | ``datagrid``     |
++-----------+------------------+
+
+The datagrid configuration is a large file that is a map of datagrid names mapped to options that
+configure their behavior:
+
+.. _reference-datagrid-columns:
+
+``actions`` (**type**: ``map``)
```

Comment:
Actually, as the order in which the options are used in the YAML file does not matter (as long as you don't use a key twice at least), I find it hard to tell which is the right order. Thus, in case two developers ordered them slightly different because they both thought their order was right, they would probably have some hard time to find the description in the reference (the order that they expect does not match the order options are actually ordered). Though when we order all options by their name, I think it's easier to find the description for one of them.